torch/jit/annotations.py

```diff
@@ -294,8 +290,10 @@ def is_tensor(ann):
 
 
 def try_ann_to_type(ann, loc):
-    if ann is None:
+    if ann is inspect.Signature.empty:
```

Comment:
What's the difference between a `Signature.empty` and a `None` (i.e. how do you get one vs the other)?

Reply:
Signature.empty implies that it is coming from functions like `def func(arg1: int)` while None implies that this is coming from functions like `def func(arg1: int) -> None`. There is no real difference in python in terms of functionality